helpers/get-conf.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+
+function get_yaml_prop() {
+    local target_file="${1}"
+    local full_key_path="${2}"
+    local default="${3}"
+
+    value=$( cat $target_file | grep -v "^#" | grep $full_key_path | tr -s [:blank:] | cut -d' ' -f 2 )
+
+    if [ $value ]
+    then
+        echo $value
+        return
+    fi
+    echo $default
```

Comment:
we can make use of the `stage-snap` package `yq`

 ```suggestion
    "${SNAP}"/bin/yq ".\"${full_key_path}\" // \"${default}\"" "${target_file}"
```

Reply:
How about adding this enhancement with the final PPA?
(I'm tring to get it working locally but I'm running into issues. Since I've just lost all my `multipass` instances, I don't have a `snapcraft` built space either to play around while it's re-building... I'm wondering not to loose time with this but concentrate on the alerts instead.)